docs/patterns/observability/logs.md

```diff
@@ -57,3 +57,24 @@ Once logs have been sent to Log Analytics, that can also be consumed by Applicat
 
 You can now interact with your logs in either of these instances.
 
+
+### Configuring Outbound Proxy Support for Fluent Bit
+If you require outbound proxy support:
+
+1. In `values.yaml`:
+    
+    a) Change `enableProxySupport` to `true`
+
+    b) Update the httpProxy and httpsProxy values to `"http://<host>:<port>"`
```

Comment:
Might be good to also add a note here that fluent-bit `1.6.4` is required for this feature to work.